.github/workflows/windows-hyperv-periodic.yml

```diff
@@ -0,0 +1,299 @@
+# Workflow intended to run containerd integration tests on Windows using Hyper-V Containers.
+
+name: Windows Hyper-V Integration Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      AZURE_SUB_ID:
+        required: true
+      AZURE_CREDS:
+        required: true
+      GCP_SERVICE_ACCOUNT:
+        required: true
+      GCP_WORKLOAD_IDENTITY_PROVIDER:
+        required: true
+
+env:
+  AZURE_DEFAULT_LOCATION: westeurope
+  AZURE_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUB_ID }}
+  AZURE_DEFAULT_VM_SIZE: Standard_D2s_v3
+  PASSWORD: Passw0rdAdmin # temp for testing, will be generated
+  DEFAULT_ADMIN_USERNAME: azureuser
+  SSH_OPTS: "-o ServerAliveInterval=20 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+  REMOTE_VM_BIN_PATH: "c:\\containerd\\bin"
+  BUSYBOX_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/busybox:1.29-2"
+  RESOURCE_CONSUMER_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/resource-consumer:1.10"
+  WEBSERVER_TESTING_IMAGE_REF: "k8s.gcr.io/e2e-test-images/nginx:1.14-2"
+  HCSSHIM_TAG: "master"
+
+
+jobs:
+  winIntegration:
+    # NOTE: the following permissions are required by `google-github-actions/auth`:
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    strategy:
+      matrix:
+        win_ver: [ltsc2019, ltsc2022]
+        include:
+        - win_ver: ltsc2019
+          AZURE_IMG: "MicrosoftWindowsServer:WindowsServer:2019-Datacenter-with-Containers-smalldisk:17763.2565.220202"
+          AZURE_RESOURCE_GROUP: ctrd-integration-ltsc2019-${{ github.run_id }}
+          GOOGLE_BUCKET: "containerd-integration/logs/windows-ltsc2019/"
+        - win_ver: ltsc2022
+          AZURE_IMG: "MicrosoftWindowsServer:WindowsServer:2022-datacenter-smalldisk-g2:20348.524.220201"
```

Comment:
We need a ```SANBOX_IMAGE``` variable here for each Windows version. For ltsc2019 it should be:

```mcr.microsoft.com/windows/nanoserver:1809``` or ```mcr.microsoft.com/windows/servercore:1809```

And for ltsc2022 it should be:

```mcr.microsoft.com/windows/nanoserver:ltsc2022```

Reply:
Added a `SANDBOX_IMAGE` entry to the testing matrix which is passed both as an env var to `make cri-integration` as well as the static containerd config used during `critest`.